### DISCUSSION TITLE
Is there a way to make sure the answer that's marked as correct appears at the top?

### DISCUSSION THREAD

**Discussion by avital** [community/community#3307] [Discussions]
In [this discussion on the Flax repo](https://github.com/google/flax/discussions/968), even when the sort order is "top", I don't see the answer that's marked as correct at the top. (This is how Stack Overflow would show it)

Is this intentional? I would think that the default view should show it this way, or at least if sorted by "top". 

cc @jheek

**Comment by thadguidry** [comment]
@avital I agree and think that's what the Top button for sorting might be used far within any individual discussion.  But it seems because GitHub devs still have work to do on the Top sort so that it indeed puts the marked answer comment at the top of the listing.

But also that the "Top" button might actually have meant to be named "Top up votes" instead and if that's the case, then GitHub devs should possibly relabel that to make it quickly understandable for anyone.
GitHub devs probably don't think that a 4th sort button is needed for "Answered", because there can only be 1 Answered marked by the original author asking the question.
Note also that the sort does not work on replies and I think that's perfectly fine and logical.

Click and see the buttons on this discussion as an example: https://github.com/ImageMagick/ImageMagick/discussions/3191?sort=top
